File: C/ptr1.c
#include <stdio.h>
int main()
 {
    int num = 10;  
    int *ptr;        
    ptr = &num;       
    printf("Value of num = %d\n", num);
    printf("Address of num = %p\n", &num);
    printf("Value using pointer = %d\n",*ptr );
    printf("Address stored in ptr = %p\n", ptr);
    return 0;
}

File: Java/ptr1.java
public class ptr1 {
    public static void main(String[] args) {
        int num = 10;   // primitive type, not reference

        // Wrapper class (Integer is an object)
        Integer obj = num;   

        System.out.println("The value of num is : " + num);
        System.out.println("The value of obj is : " + obj);
        System.out.println("The hashCode (like address) of obj is : " + obj.hashCode());
    }
}
